tests/appsec/test_remoteconfiguration.py

```diff
@@ -1,16 +1,25 @@
 import os
 
+import mock
 import pytest
 
 from ddtrace.appsec._remoteconfiguration import appsec_rc_reload_features
+from ddtrace.appsec.utils import _appsec_rc_capabilities
 from ddtrace.appsec.utils import _appsec_rc_features_is_enabled
 from ddtrace.constants import APPSEC_ENV
 from ddtrace.constants import APPSEC_JSON
 from ddtrace.contrib.trace_utils import set_http_meta
 from ddtrace.ext import SpanTypes
+from ddtrace.internal.remoteconfig import RemoteConfig
 from tests.utils import override_env
 
 
+@pytest.fixture
+def remote_config_worker(tracer):
+    RemoteConfig._worker = None
```

Comment:
```suggestion
    # Make sure to start with the state we care about
    if RemoteConfig._worker:
        RemoteConfig._worker.stop()
        RemoteConfig._worker = None
    try:        
        yield
    finally:
         # Cleanup when we are done.
         RemoteConfig._worker.stop()
         RemoteConfig._worker = None
```

I would also suggest moving this to `tests/appsec/conftest.py` so it is available for all tests to use, it seems useful

Reply:
This seems to work well, on next commit:

```
def _stop_remote_config_worker():
    if RemoteConfig._worker:
        RemoteConfig._worker._stop_service()
        RemoteConfig._worker = None


@pytest.fixture
def remote_config_worker(tracer):
    _stop_remote_config_worker()
    try:
        yield
    finally:
        _stop_remote_config_worker()
```